src/Console/ConsoleOptionParser.php

```diff
@@ -576,6 +576,7 @@ public function addSubcommand($name, array $options = [])
             $command = $name;
             $name = $command->name();
         } else {
+            $name = Inflector::underscore($name);
```

Comment:
Next goofy scenario I would be worried about is. If both `initMyDb` and init_my_db` are proper sub-commands. Right now only one will be callable, but previously both would be. Is that perhaps a scenario that should be rare enough to not worry about? Should we trigger an error/exception when duplicate methods are attached?